lib/pbench/test/unit/server/test_requests.py

```diff
@@ -126,6 +126,13 @@ def test_missing_authorization_header(self, client, caplog, server_config):
         self.verify_logs(caplog)
 
     def test_malformed_authorization_header(self, client, caplog, server_config):
+        response = client.put(
```

Comment:
Yeah; you did a good job on factoring out the common `query_api` tests into `Commons`, and we really need to think about repeating that out to the non-Elasticsearch-based resource level, akin to `ApiBase`. But not today ...